bugwarrior/services/bitbucket.py

```diff
@@ -199,20 +200,20 @@ def get_owner(self, issue):
     def issues(self):
         user = self.config.get(self.target, 'bitbucket.username')
         response = self.get_collection('/repositories/' + user + '/')
-        repo_tags = filter(self.filter_repos, [
+        repo_tags = list(filter(self.filter_repos, [
```

Comment:
It's used twice in the same method, in py3 the second time it's "empty".  The following snippet rais an AssertionError in py3, while it succeed in py2:
```
a = [1, 2, 3, 4, 5]
b = filter(lambda x: True, a)
c = [el for el in b]
d = [el for el in b]
assert len(c) == len(d)
```
I think futurize is right here :-)